lms/djangoapps/course_goals/templates/course_goals/edx_ace/goalreminder/email/body.html

```diff
@@ -8,6 +8,7 @@
         font-size: 1em;
         line-height: 1.5;
         max-width: 600px;
+        border: 10px solid white;
```

Comment:
padding/margin didn't seem to work here but a border seems to work :/

Reply:
How does this even work if all the styles are inline? table_style doesn't appear to be referenced anywhere.

Reply:
ok. CSS is fun!

Reply:
this is a django template which extends the base_body template and table_style is referenced there